chrome/content/dkim.js

```diff
@@ -125,19 +125,50 @@ DKIM_Verifier.Display = (function() {
 
 	/**
 	 * Sets the url to the favicon. Empty string to reset it.
-	 * 
+	 *
 	 * @param {String} faviconUrl
 	 * @return {void}
 	 */
 	function setFaviconUrl(faviconUrl) {
-		expandedFromBox.dkimFaviconUrl = faviconUrl;
 
-		// for CompactHeader addon
-		if (collapsed1LfromBox) {
-			collapsed1LfromBox.dkimFaviconUrl = faviconUrl;
-		}
-		if (collapsed2LfromBox) {
-			collapsed2LfromBox.dkimFaviconUrl = faviconUrl;
+		let activeTheme = Services.prefs.getBranch("general.skins.").getCharPref("selectedSkin");
+
+		if (activeTheme && activeTheme.match(/^silvermel|charamel$/i)) {
```

Comment:
Did you perhaps mean `match(/^(?:silvermel|charamel)$/i)`?

Currently `silvermel foo` and `foo charamel` are matched too.

Reply:
Yes, this exactly, what I meant. Thank you for the catch.